nifi-framework-bundle/nifi-framework/nifi-framework-core-api/src/main/java/org/apache/nifi/groups/BundleUpdateStrategy.java

```diff
@@ -22,11 +22,7 @@ public enum BundleUpdateStrategy {
     /**
      * If a proposed flow has a different bundle than the current flow, the bundle is not updated
      */
-    IGNORE_BUNDLE,
```

Comment:
This could be valid in some circumstances and should be reverted.

Reply:
Can you share more about how this would be valid but never referenced?

Reply:
On closer review, this appears to be limited to explicit usage in method arguments, so it seems like it could be removed. However, as a framework-core-api element, it may be better considered in the context of other framework changes, keeping it for the moment seems best.